src/Psl/Internal/suppress.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Internal;
```

Comment:
Any reason why you want to internalize this one?
It looks like a function I could use in some of the packages I write as well.

Together with a shortcut function for:

```php
$result = some_php_function_that_could_return_false();
if ($result === false) {
    $err = error_get_last();

    Psl\invariant_violation('error(%d): %s.', $err['type'], $err['message']);
}
```

Reply:
because this is a "hack" around a behavior that is only caused by PHP builtin functions, PSL functions don't trigger any errors, only exceptions, if you are doing this as well, you are calling php builtin functions, which PSL is trying to replace, so instead of providing a way for people to work around PHP builtin functions, we should provide then with a complete *sane* replacement, that's why i'm against exposing things like this in the public API;

Reply:
Agree with that vision.
Nothing is really internal though :)